examples/esp_idf/lightdb_stream/pytest/test_sample.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024 Golioth, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import contextlib
+import logging
+import pytest
+import time
+import yaml
+import datetime
+import re
+
+LOGGER = logging.getLogger(__name__)
+
+pytestmark = pytest.mark.anyio
+
+async def test_lightdb_stream(board, device):
+    time.sleep(2)
+
+    # Set Golioth credential
+    golioth_cred = (await device.credentials.list())[0]
+    board.set_golioth_psk_credentials(golioth_cred.identity, golioth_cred.key)
+
+    # Reset board
+    board.reset()
```

Comment:
We can remove this now that https://github.com/golioth/golioth-firmware-sdk/pull/385 is merged.